src/queries.js

```diff
@@ -69,13 +69,19 @@ function queryByLabelText(...args) {
 function queryAllByText(
   container,
   text,
-  {selector = '*', exact = true, collapseWhitespace = true, trim = true} = {},
+  {
+    selector = '*',
+    exact = true,
+    collapseWhitespace = true,
+    trim = true,
+    ignore = 'script',
```

Comment:
I think script _and style_ would be a better default.

Reply:
Good idea. I'll add that!

Reply:
But I'm just a couple minutes late...

Reply:
No worries. I just pushed an update directly to master. I figured the overhead would be unnecessary.

Reply:
Great :+1: